webpack/webpack.common.js

```diff
@@ -17,6 +17,12 @@ module.exports = opts => {
 				PRODUCTION: !opts.devMode,
 			}),
 
+			new webpack.DefinePlugin({
+				VPXJS_RELEASE_VERSION: JSON.stringify(
+				  require('../node_modules/vpx-js/package.json').version
+				),
+			  }),
+

```

Comment:
There's already `webpack.DefinePlugin` just above, can you just add it there? :)

Reply:
sure